policy/sample_points/lsst_gaussian_quadrature_points.yaml

```diff
@@ -0,0 +1,42 @@
+---
+
+# This file stores the Gauss-Legendre quadrature points for the LSST field
```

Comment:
Gauss-Legendre quadrature -> Gauss-Legendre quadrature (GQ).

This applies to all the `yaml` files.